Cargo.toml

```diff
@@ -34,6 +34,11 @@ flate2 = "1.0.7"
 [dev-dependencies]
 assert_cmd = "0.11.1"
 fs_extra = "1.1.0"
+pre-commit = "0.5.2"
+
+[package.metadata.precommit]
+fmt = "cargo fmt"
+clippy = "cargo clippy"
```

Comment:
If we want other hooks (or if we want to add some arguments to these commands) we can add them easily here. I was initially using `cargo-husky` but their user-hooks implementation wasn't working

Reply:
`precommit` package isn't compatible w/Windows. Trying `rusty-hook`